src/components/CardForm.tsx

```diff
@@ -26,10 +27,24 @@ export const CardForm = ({ columnID }: Props) => {
 
   const addNewCard = (text: string) => {
     if (text.trim().length > 0) {
-      firestore.collection('users').doc(uid).collection('cards').add({
-        text,
-        columnID,
-      })
+      firestore
+        .collection('users')
+        .doc(uid)
+        .collection('cards')
+        .add({
+          text,
+        })
+        .then((docRef) => {
+          firestore
+            .collection('users')
+            .doc(uid)
+            .collection('columns')
+            .doc(columnId)
+            .update({
+              cardsIds: [...cardsIds, docRef.id],
+            })
+        })
+
```

Comment:
```suggestion
      const userRef = firestore.collection('users').doc(uid)
      const cardRef = await userRef.collection('cards').add({
        text,
      })

      const columnRef = userRef.collection('columns').doc(columnId)
      columnRef.update({
        cardsIds: [...cardsIds, cardRef.id],
      })

```
A lot cleaner, don't you think?

Reply:
You can eventually do the same in `src/components/ColumnForm.tsx`

Reply:
Maybe a little cleaner :)